src/Components/SplashPage/SplashPage.js

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.section`
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+const SplashPage = () => {
+  return (
+    <Wrapper>
+      <h1>
+        Welcome to Drag Time!
+      </h1>
+      <NavLink to='/home'>Go to my events</NavLink>
```

Comment:
We might consider changing this to "See Events" since I this will take you to the all events screen

Reply:
Yeah we just put this in there as a current filler - I was thinking something along the lines of just "Enter" or "Enter Site" but "See Events" would also work!